src/main/java/Example.java

```diff
@@ -59,8 +59,11 @@ public static void main(final String[] args) throws IOException, InterruptedExce
             try {
                 client.indices().create(createIndexRequest);
 
-                // add settings to the index
-                IndexSettings indexSettings = new IndexSettings.Builder().build();
+                // Update index settings
+                IndexSettings indexSettings = new IndexSettings.Builder()
+                    .refreshInterval(b -> b.time("2s"))
```

Comment:
Does this exist on serverless? If not comment it out.

Reply:
It was failing on OpenSearch domain with an error stating that there is no IndexSetting to apply (since it is just an empty setting). May be version specific, I only tested on the latest `2.13` (OpenSearch_2_13_R20240520-P1).

Reply:
Just checked this with an OpenSearch serverless instance and it works fine.

Reply:
Removed this (was a separate commit) to ensure nothing is broken.